spinnaker/main.tf

```diff
@@ -160,6 +160,7 @@ module "spinnaker_dns" {
   x509_ip_addresses  = data.terraform_remote_state.static_ips.outputs.api_x509_ips_map
   vault_ip_addresses = data.terraform_remote_state.static_ips.outputs.vault_ips_map
   ship_plans         = data.terraform_remote_state.static_ips.outputs.ship_plans
+  grafana_ip_addresses = data.terraform_remote_state.static_ips.outputs.grafana_ips_map
```

Comment:
Inside the spinnaker directory, you would need to run `terraform fmt -recursive`